generators/app/index.js

```diff
@@ -35,7 +29,7 @@ class GeneratorReact extends Generator {
     return Promise.resolve().then(() => {
       this.log('Copying base project files...');
       configPackageJson.bind(this)();
-      copyTemplateFiles.bind(this)();
+      copyTemplateFiles.bind(this)(this.projectName);
```

Comment:
Its not necessary to pass that argument, since that function is getting the project name from the binded object `this`